xbmc/settings/VideoSettings.h

```diff
@@ -61,8 +61,13 @@ class CVideoSettings
   bool m_StereoInvert;
   int m_VideoStream;
 
-  static int GetNextViewMode(int viewMode);
-  static int GetViewModeStringIndex(int viewMode);
+  typedef struct
+  {
+    const int stringIndex;
+    const int nextViewMode;
```

Comment:
"next" represents the next view mode to display in the GUI. The enumeration order does not map well to the order in which the modes should be displayed in the GUI, IMO. The enumeration order also cannot be changed since the enumeration value is used when stored in the settings. Changing them would mess up everyone's view mode settings.